optimade/models/structures.py

```diff
@@ -876,66 +889,64 @@ def validate_structure_features(cls, v, values):
             raise ValueError(
                 f"structure_features MUST be sorted alphabetically, given value: {v}"
             )
-        # disorder
-        for species in values.get("species", []):
-            if len(species.chemical_symbols) > 1:
-                if StructureFeatures.DISORDER not in v:
-                    raise ValueError(
-                        f"{StructureFeatures.DISORDER.value} MUST be present when any one entry in species "
-                        "has a chemical_symbols list greater than one element"
-                    )
-                break
-        else:
-            if StructureFeatures.DISORDER in v:
-                raise ValueError(
-                    f"{StructureFeatures.DISORDER.value} MUST NOT be present, since all species' chemical_symbols "
-                    "lists are equal to or less than one element"
-                )
+
         # assemblies
-        if values.get("assemblies", None) is not None:
+        if values.get("assemblies") is not None:
             if StructureFeatures.ASSEMBLIES not in v:
                 raise ValueError(
                     f"{StructureFeatures.ASSEMBLIES.value} MUST be present, since the property of the same name is present"
                 )
-        else:
-            if StructureFeatures.ASSEMBLIES in v:
-                raise ValueError(
-                    f"{StructureFeatures.ASSEMBLIES.value} MUST NOT be present, "
-                    "since the property of the same name is not present"
-                )
```

Comment:
A side effect here is that we can no longer do any validation that has to check for "MUST NOT be present" restrictions

Reply:
How come, exactly? We should still be able to check whether a key is present or not?

Reply:
We can, but we can't make any claim about *why* the key is not present, as it's now valid for it to be not present for reasons other than "there are no assemblies". (I say this below in a more waffley way...)